vimeo-networking/src/main/java/com/vimeo/networking/model/PlatformConstraint.java

```diff
@@ -0,0 +1,62 @@
+package com.vimeo.networking.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.stag.UseStag;
+
+import java.io.Serializable;
+
+/**
+ * Constraints put in place by a social media platform on uploading videos.
+ */
+@SuppressWarnings({"WeakerAccess", "unused"})
+@UseStag
+public class PlatformConstraint implements Serializable {
+
+    private static final long serialVersionUID = -3004640141349461475L;
+
+    @SerializedName("duration")
+    private int mDuration;
+
+    @SerializedName("size")
+    private Long mSize;
```

Comment:
Should be consistent here and either use `long` primitive for `mSize` or `Integer` class for `mDuration`.